src/components/breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import _ from 'lodash';
+
+import './breadcrumbs.css';
+
+export default function Breadcrumbs(props) {
+  const {breadcrumbs}=props;
+
+  const breadcrumbsList = _.map(breadcrumbs, (breadcrumb)=> {
+    const { link, title} = breadcrumb;
+
+    if (link && title) {
```

Comment:
nope, use react-router to detect current path and compare w/ it or use additional key like `active: true`